lib/ecto/changeset.ex

```diff
@@ -1464,6 +1464,8 @@ defmodule Ecto.Changeset do
   ## Options
 
     * `:message` - the message on failure, defaults to "can't be blank"
+    * `:trim` - boolean, sets whether leading whitespaces are removed
+                before running the validation, defaults to true
```

Comment:
Looking at the codebase, it seems "Options" are indented differently:

```
  * `:trim` - boolean, sets whether leading whitespaces are removed
    before running the validation, defaults to true
```

Check https://github.com/elixir-ecto/ecto/blob/master/lib/ecto/changeset.ex#L390-L391